api/defaults.go

```diff
@@ -346,13 +350,13 @@ func defaultNodeConfig(installationID string, request *requests.CreateAccount, o
 	if request.VerifyTransactionURL != nil {
 		nodeConfig.ShhextConfig.VerifyTransactionURL = *request.VerifyTransactionURL
 	} else {
-		nodeConfig.ShhextConfig.VerifyTransactionURL = defaultNetworks[0].FallbackURL
+		nodeConfig.ShhextConfig.VerifyTransactionURL = mainnet(request.WalletSecretsConfig.StatusProxyStageName).FallbackURL
```

Comment:
This is not using our proxy, should i set it to the proxy?

Reply:
Same thing here, what is actually needed? Apparently the URL to whatever network we have first on the list, which happens to be eth mainnet. Don't the "verify transactions" on other chains?

Reply:
Anyway, better leave it as it is for now, but it's definitely something we want to look into.

Reply:
no, we only verify on mainnet, becaues on the frontend we don't differentiate chat transactions from different networks

Reply:
I see! We probably want to make those parts of the code use the same Proxy->Grove->Infura fallback mechanism used everywhere else though. So instead of the URL they should be getting a reference to the RPC Client. Not a change we want to do now though.